include/openmc/distribution.h

```diff
@@ -6,12 +6,11 @@
 
 #include <cstddef> // for size_t
 
-#include "pugixml.hpp"
-#include <gsl/gsl-lite.hpp>
-
 #include "openmc/constants.h"
 #include "openmc/memory.h" // for unique_ptr
 #include "openmc/vector.h" // for vector
+#include "pugixml.hpp"
+#include <gsl/gsl-lite.hpp>
```

Comment:
You may have misunderstood my earlier comment- I think these should stay where they originally were, and also with the blank line separating them from the others